bindings/pyroot/cppyy/CPyCppyy/src/Pythonize.cxx

```diff
@@ -1157,6 +1167,7 @@ bool CPyCppyy::Pythonize(PyObject* pyclass, const std::string& name)
         Utility::AddToClass(pyclass, "__cmp__",  (PyCFunction)StlStringCompare, METH_O);
         Utility::AddToClass(pyclass, "__eq__",   (PyCFunction)StlStringIsEqual, METH_O);
         Utility::AddToClass(pyclass, "__ne__",   (PyCFunction)StlStringIsNotEqual, METH_O);
+        ((PyTypeObject*)pyclass)->tp_hash = (hashfunc)StlStringHash;
```

Comment:
We could also implement `__hash__` in Python, or expose it here via the `AddToClass` utility.

Reply:
Answered by the PR description, already seen by @stwunsch !